guides/schema/mutations.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Schema — Mutations
+---
+
+## Mutations
+
+Registering a mutation root allows to define fields that can mutate your data.
+
+```ruby
+Schema = GraphQL::Schema.define do
+  query QueryRoot
+  mutation MutationRoot
+end
+
+MutationRoot = GraphQL::ObjectType.define do
+  name "Mutation"
+
+  field :addPost, Post do
+    description "Adds a Post."
+
+    # Use Input Types to define complex argument types
+    argument :post, PostInputType
+    resolve ->(t, args, c) {
+      title = args['post']['title']
+      description = args['post']['description']
+      Post.create(title: title, description: description)
+    }
+  end
+end
+
+PostInputType = GraphQL::InputObjectType.define do
+  name "PostInputType"
+  description "Properties for creating a Post"
+
+  argument :title, !types.String do
+    description "Title of the post."
+  end
+
+  argument :description, types.String do
+    description "Description of the post."
+  end
+end
+```
+
+## Nested mutations
```

Comment:
Yes, I was confused by this too. I thought that "nested mutations" referred to this: 

https://github.com/facebook/graphql/issues/252

(Which is the same as the example posted above.)

I would have described this as "mutations with complex inputs". However, I may be wrong to think that since at least the Graph.cool folks _have_ called this "nested mutations": https://blog.graph.cool/improving-dx-with-nested-mutations-af7711113d6#.syi1rk3o8